src/guides/manage-installation.rst

```diff
@@ -0,0 +1,119 @@
+==================================
+Managing a Nextstrain Installation
+==================================
+
+.. contents::
+   :local:
+
```

Comment:
Added an intro section in [force-push](https://github.com/nextstrain/docs.nextstrain.org/compare/427b38c5be7d9649469743731f01a57d3ed9a4b4..4366c1b002d7f78291ce281cb478fa45b53ed886).

> it doesn't actually tell you how to find out which runtime you are using

@tsibley is `nextstrain check-setup` the best way to determine runtime(s) that have been set up? I noticed that its purpose is slightly different: it lists the currently supported runtimes rather than all runtimes that have been set up. For example, I have the Docker runtime set up but it does not appear in the list since my docker daemon is not running.

Reply:
Not really, no.

"the runtime you are using" means "the default runtime", at least in the absence of the runtime selection flags like `--docker`, `--conda`, etc. The default runtime is determined by the `core.runner` value in the config file, with a fallback to `docker` if there's no value (or it's invalid).

There's not a clean user-friendly way of inspecting runtimes/config. This is part of what I think should be tackled when I say "I've been wanting to more fully document and compare/contrast the runtimes for a bit". But in the meantime, `nextstrain version --verbose` will report on each runtime and indicate the default with `(default)` after the name; it includes a lot of other stuff too, however.

```
$ nextstrain version --verbose
nextstrain.cli 6.2.1

Python
  /home/tom/.nextstrain/cli-standalone/nextstrain
  3.10.9 (main, Dec 21 2022, 04:02:04) [Clang 14.0.3 ]

Runners
  docker (default)
    nextstrain/base:build-20230510T204858Z (3122df6b3a80, 2023-05-08 15:30:15 -0700 PDT)
    augur 21.1.0
    auspice v2.45.2
    fauna ca21fbc
    sacra not present

  conda 
    nextstrain-base 20230314T174809Z (hb0f4dca_1_locked, nextstrain)
    augur 21.1.0
    auspice 2.45.0

  singularity 
open /nextstrain//.GIT_ID: no such file or directory
open /nextstrain//.GIT_ID: no such file or directory
    docker://nextstrain/base:branch-trs-disable-provenance (/home/tom/.nextstrain/runtimes/singularity/images/nextstrain/base/branch-trs-disable-provenance.sif)
    augur 21.1.0
    auspice v2.45.2
    fauna 3b3d01f
    sacra not present

  ambient 
    unknown

  aws-batch 
    unknown
```

Reply:
I see. Given the current limitations, how about having something like this for now:

> Steps vary by runtime option (Docker, Conda, ambient). To determine which Nextstrain runtime you are using, run:
> 
> ```
> nextstrain version --verbose
> ```
> 
> and look for the name that has `(default)` next to it.

Then in the future it can be simplified to one command that concisely shows the default runtime.

Reply:
Sounds fine to me.

Reply:
Updated in [force-push](https://github.com/nextstrain/docs.nextstrain.org/compare/4366c1b002d7f78291ce281cb478fa45b53ed886..bb947726995254ea685c71e561b042c527bc982b).